tests/beman/inplace_vector/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ add_gtest(compare)
 add_gtest(constructors)
 add_gtest(size_n_data)
 add_gtest(erasure)
+add_gtest(modifiers)
```

Comment:
I don't know why we did not caught this. I just realized this as well, this is embarrassing lol

Reply:
It's often difficult to 'see a negative'.  Also this implementation hasn't necessarily had a consistent group working on it...